### PR TITLE
Rotates the eth2 logs

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

https://github.com/obscuronet/obscuro-internal/issues/1441

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

**Review by StefanIliev545 — APPROVED** [review]
LGTM, but I'd prefer if we used some library for this one

**Comment by otherview** [comment]
> LGTM, but I'd prefer if we used some library for this one

Yeah my thoughts too, but I didn't see anything that was just right. Everything I saw was either bloated or deprecated :(
Any lib you might suggest ?

File: integration/eth2network/rotation_writer.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		if err != nil {
+			fmt.Println(err)
+		}
+	}
+
+	w.currentSize = 0
+	w.numFilesRotated = 0
+
+	// Construct the file name for the next log file
+	timestamp := time.Now().Format("2006-01-02T15-04-05")
```
Comment (moraygrieve) [inline integration/eth2network/rotation_writer.go:51]: My ignorance of go - but is that the correct format specifier?

Comment (otherview) [reply integration/eth2network/rotation_writer.go:51]: Yeah it is a weird format approach, I'm not really sure of the story behind it, but it is.

**Review by moraygrieve — APPROVED** [review]

**Pull request merged**